src/Grid/Column/Resolver/DatetimeResolver.php

```diff
@@ -0,0 +1,61 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Grid\Column\Resolver;
+
+use Carbon\Carbon;
+use Pimcore\Bundle\StudioBackendBundle\Exception\Api\InvalidArgumentException;
+use Pimcore\Bundle\StudioBackendBundle\Grid\Column\ColumnResolverInterface;
+use Pimcore\Bundle\StudioBackendBundle\Grid\Schema\Column;
+use Pimcore\Bundle\StudioBackendBundle\Grid\Schema\ColumnData;
+use Pimcore\Bundle\StudioBackendBundle\Grid\Util\Trait\ColumnDataTrait;
+use Pimcore\Bundle\StudioBackendBundle\Grid\Util\Trait\SimpleGetterTrait;
+use Pimcore\Bundle\StudioBackendBundle\Util\Constants\ElementTypes;
+use Pimcore\Model\Element\ElementInterface;
+
+/**
+ * @internal
+ */
+final class DatetimeResolver implements ColumnResolverInterface
+{
+    use SimpleGetterTrait;
+    use ColumnDataTrait;
+
+    /**
+     * @throws InvalidArgumentException
+     */
```

Comment:
Isn't Carbon::create throwing InvalidFormatException?

And what about handling the Exception here?  Like return null as data? WDYT?

Reply:
Exception is coming from the SimpleGetter Trait, since if the field does not exist, it throws the invalidargument exception.
Same goes for all the other resolvers

Reply:
In general we should maybe create a follow up ticket and talk to the frontend on how to deliver dates

Reply:
But then we should add the InvalidFormatException as well here?

Reply:
Maybe we should just deliver the timestamp for now, not make things more complicated than they need to be, if we do not know how frontend will handle it anyways :)

Reply:
Works for me :) Sounds Good :+1: